### ISSUE TITLE
Elegant fail

### ISSUE THREAD

**Issue by shahpnmlab** [eisfabian/PACEtomo#3]
Hi Fabian, 
We managed to collect 532 TS's in 48hrs with your script! However, I was wondering if there would be a way for the script to fail more elegantly? The specific scenario that I encountered during data collection was when the estimated IS was beyond the limit of the scope (we were shooting a 7x7 pattern)and the script just stopped. Thus, we ended up with some downtime. So the suggestion then would be, for the script to check of the IS is within range if not, skip that hole position and move on to next. Do you think this would be possible to do?
Best,
pranav

**Comment by eisfabian** [comment]
Hi Pranav,

Wow! Enjoy processing this data!
I agree, this could be prevented. For now, it might help to change the imageShiftLimit to 25, but I will include a check for the IS and possibly for black images in future versions! I hope I get around to it some time soon!

Thanks for the feedback!
Best,
Fabian

**Comment by eisfabian** [comment]
Hi Pranav,

I uploaded a new version to the beta folder. It includes a check if the IS is at the SerialEM image shift limit minus wiggle room for alignment. If it is at the limit it will skip the affected branch for the rest of the acquisition. Additionally I included an optional setting to do the same for dark images. I hope this catches cases like yours but I have not extensively tested it yet!

Thanks again for your feedback!
Best,
Fabian
